config/ignition_fortress_ubuntu_focal.yaml

```diff
@@ -6,9 +6,6 @@ architectures: [amd64, armhf, arm64, source]
 filter_formula: "\
 (Package (= ignition-fortress) \
 ), $Version (% 1.0.3-1*) |\
-(Package (= ignition-cmake2) |\
- Package (= libignition-cmake2-dev) \
-), $Version (% 2.12.1-1*) |\
```

Comment:
I would prefer that these changes are revered for the reasons described in https://github.com/ros-infrastructure/reprepro-updater/pull/168#issuecomment-1218663623

Reply:
Reverted [adf0dce](https://github.com/ros-infrastructure/reprepro-updater/pull/168/commits/adf0dcee76545baaff671047521d91dad91e8176)